pkg/cmd/cli/cmd/deploy.go

```diff
@@ -26,6 +26,12 @@ NOTE: This command is still under active development and is subject to change.`
 	deploy_example = `  // Display the latest deployment for the 'database' deployment config
   $ %[1]s deploy database
 
+  // List all deployments for the 'database' deployment config
+  $ %[1]s deploy database -L
```

Comment:
So why wouldn't I just describe dc/database?  What does this add that the other does not?